src/main/resources/default.properties

```diff
@@ -0,0 +1,165 @@
+# app name
```

Comment:
That was my idea. Have an explicit `defaults.properties` which is used first. Overwrite that with anything in a classpath `portal.properties`, which folks can bake in as a war overlay. And finally, overwrite that with anything from some external property source. Spring should follow a similar pattern, but I haven't done that yet. In effect, this means the `.EXAMPLE` file can be actually used as fallbacks rather than copied and edited.